File: GptFactChecker/gpt-fact-checker-front/src/model/Claim.ts
import type ClaimCheck from "./ClaimCheck";

class Claim {
	public id?: string;
	public claimSummarized: string;
	public claimRawText: string;
	public tags: string[];
	public claimChecks?: ClaimCheck[];

	constructor() {
		this.claimSummarized = "";
		this.claimRawText = "";
		this.tags = [];
	}
}

export default Claim;

File: GptFactChecker/GptFactCheckerApi/Model/Claim.cs
using System.Security.Cryptography.X509Certificates;

namespace GptFactCheckerApi.Model;

public class Claim
{
    public string Id { get; set; }
    public string ClaimSummarized { get; set; }
    public string ClaimRawText { get; set; }
    public string[] Tags { get; set; }
}
